ibdgc.datacommons.io/portal/gitops.json

```diff
@@ -116,6 +116,10 @@
     },
     "topBar": {
       "items": [
+        {
+          "link": "https://ibd-browser.planx-pla.net/",
```

Comment:
no `.datacommons.io` prod link?

Reply:
Yeah. This is running inside a stand alone vm and that's the url where the loadbalancer is pointing to. Just for demo purpose. So it would be either planx-pla.net or datacommons.io 🤷

Reply:
does it get killed every night? if it's prod ready should we create a release and deploy to a prod vm?

Reply:
nope this vm is not killed since this is totally separate from gen3 like covid19 app. It has it's own vpc, elb which I created manually. We are not using kubernetes or any technology that gen3 is using for this app for now. However It will be in gen3 which I am working on but not for this demo site. I am adding it because yesterday Phil wanted us to have a link on ibd home page to show people during the presentation. If we want to use datacommons.io, i can flip over the certificate for both qa-ibd and prod-ibd

Reply:
i think it'd make more sense to use datacommons.io since it will be available in production but sounds like it's not urgent

Reply:
cool thanks! Only gdc team has access to datacommons.io. I will merge this pr first and work with them on the new cert.